source/deployment-options/docker/upgrading-wazuh-docker.rst

```diff
@@ -22,6 +22,11 @@ Using default docker-compose files
 
       # docker-compose down
 
+#. Update the local git repository with the current versions of wazuh-docker:
+
+      .. code-block::
+
+         # git fetch
```

Comment:
Hello @cladkins

I agree with this change, as it is necessary to enter a git command to update the repository and be able to checkout the tag, if not, it gives the following error:

```
error: pathspec 'branch' did not match any file(s) known to git
```

But before accepting these changes, I need to ask you for other changes:

1. Create an issue explaining the problem and that you have created this PR to solve it since every PR must be linked to an issue, it doesn't need to be large, just the issue you found and the proposed solution, also, please add a small description to the pull request description section
2. Modify your changes as follows
2.1 Fix the indentation so that it follows the established indentation (the `git checkout` is also wrong, if you can fix it in the process I would appreciate it) (blue line) Note that this must include the `.. code-block::` and the `# git...` lines
2.2 Add a line break to separate `git fetch` from the next point (red line)

![image](https://github.com/wazuh/wazuh-documentation/assets/14913942/70fe2f8e-3fec-4052-ac05-2b668a86850f)